src/plugins/kibana_utils/public/state_management/url/kbn_url_tracker.ts

```diff
@@ -58,6 +58,12 @@ export function createKbnUrlTracker({
   toastNotifications,
   history,
   storage,
+  shouldTrackUrlUpdate = pathname => {
+    const currentAppName = defaultSubUrl.slice(2); // cut hash and slash symbols
```

Comment:
Makes me realize that `kbn_url_tracker` works only for current Kibana's url format and hash router. Will need to revisit when / if someone wants to use it in np app with browser router

Reply:
That's true, I didn't want to introduce too many degrees on freedom on this helper just yet to keep the complexity low. Let's generalize once there are use cases for this